src/components/DropArea/components/SbDropAreaUpload.vue

```diff
@@ -0,0 +1,114 @@
+<template>
+  <SbBlockUi>
+    <div class="sb-drop-area__upload">
+      <SbIcon
+        name="refresh"
+        size="small"
+        color="primary"
+      />
+      <span class="sb-drop-area__upload-label">
+        {{ returnLabelToUpload }}
+      </span>
+      <span class="sb-drop-area__time-left">
+        {{ returnTimeLeft }}
+      </span>
+      <SbLoading
+        type="bar"
+        :value="percentageValue"
+      />
+    </div>
+  </SbBlockUi>
+</template>
+
+<script>
+import SbIcon from '../../Icon'
+import SbLoading from '../../Loading'
+import SbBlockUi from '../../BlockUI'
+
+export default {
+  name: 'SbDropAreaUpload',
+
+  components: {
+    SbIcon,
+    SbLoading,
+    SbBlockUi
+  },
+
+  props: {
+    totalFiles: {
+      type: [String, Number],
+      default: 0
+    },
+    actualFile: {
+      type: [String, Number],
+      default: 0
+    },
+    fileName: {
+      type: String,
+      default: null
+    },
+    percentageValue: {
+      type: [String, Number],
+      default: 0
+    },
+    timeLeft: {
+      type: [String, Number],
+      default: 0
+    }
+  },
+
+  computed: {
+    returnLabelToUpload () {
+      return `Uploading ${this.actualFile}/${this.totalFiles} - ${this.fileName}`
+    },
+    returnTimeLeft () {
+      return `${this.timeLeft} sec left`
```

Comment:
I believe that it's better to receive a time in seconds and we handle with this number showing the correct time unit. Examples:

* If the user pass 40 as `timeLeft`, you could show: "40 sec left"
* If the user pass 90 as `timeLeft`, you could show: "1 min and 30 sec left"

And so on

Reply:
I agree!